pkg/harvester/edit/harvesterhci.io.addon/rancher-monitoring.vue

```diff
@@ -204,7 +204,7 @@ export default {
   <Tabbed :side-tabs="true">
     <Tab
       name="basic"
-      :label="t('generic.basic')"
```

Comment:
Adding a lot of `basic` translations is not a good way to do it, it is a generic translation.  
I'll merge this pr first, because the problem exists elsewhere, and we should submit a pr to unify the solution.